pages/landing/patcher/use-cases/index.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+extra_nav_links:
+  - name: Use Cases
+    url: /patcher/use-cases/
+
+title: Patcher Use Cases
+excerpt: Some examples of scenarios where Patcher can be useful
+permalink: /patcher/use-cases/
+
+use_cases:
+  - heading: "Use case 1: Update to Terraform 1.0 automatically"
+    tab_block: site.data.patcher-landing-page-tabs-terraform-upgrade
+    body: |
+      <ul>
+        <li>Use Patcher to update your Terraform code from an older version (e.g. 0.11) all the way up to the latest version (e.g. 1.0) automatically.</li>
+        <li>Patcher does this using <i>Patch Files</i>, which transform code automatically (e.g. switching from HCL1 to HCL2 syntax or updating <code>source</code> URLs to compatible module versions) and open pull requests with the results.</li>
+        <li>You can use off-the-shelf Patch Files created by Gruntwork (e.g. we offer Terraform upgrade patches such as <code>tf-13-to-14.patch</code>, <code>tf-14-to-15.patch</code>, etc) or create your own Patch Files!</li>
+      </ul>
+
+  - heading: "Use case 2: Update to React 17.0 automatically"
+    image: data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9Ii0xMS41IC0xMC4yMzE3NCAyMyAyMC40NjM0OCI+CiAgPHRpdGxlPlJlYWN0IExvZ288L3RpdGxlPgogIDxjaXJjbGUgY3g9IjAiIGN5PSIwIiByPSIyLjA1IiBmaWxsPSIjNjFkYWZiIi8+CiAgPGcgc3Ryb2tlPSIjNjFkYWZiIiBzdHJva2Utd2lkdGg9IjEiIGZpbGw9Im5vbmUiPgogICAgPGVsbGlwc2Ugcng9IjExIiByeT0iNC4yIi8+CiAgICA8ZWxsaXBzZSByeD0iMTEiIHJ5PSI0LjIiIHRyYW5zZm9ybT0icm90YXRlKDYwKSIvPgogICAgPGVsbGlwc2Ugcng9IjExIiByeT0iNC4yIiB0cmFuc2Zvcm09InJvdGF0ZSgxMjApIi8+CiAgPC9nPgo8L3N2Zz4K
+    image_side: left
+    image_width: 379px
+    body: |
+      <ul>
+        <li><a href="https://reactjs.org/blog/2020/08/10/react-v17-rc.html">React 17.0</a> is a major release that had a number of breaking changes. As an example, one of the breaking changes in this release is that returning <code>undefined</code> from <code>forwardRef</code> and <code>memo</code> components is an error and you should return <code>null</code> instead.</li>
+        <li>Use Patcher to update your code to React 17.0, by applying a <i>Patch File</i> that, among other things, ensures that <code>forwardRef</code> and <code>memo</code> components never return <code>undefined</code>.</li>
+      </ul>
+
+  - heading: "Use case 3: Update to Ruby on Rails 2.3.16/3.0.20 automatically"
```

Comment:
It's important in this one to highlight that it's not just a version bump, _but a way to patch your code automatically to fix a security vulnerability._ So perhaps the heading should be, "Use case 3: Automatically patch a severe security vulnerability in Ruby on Rails."

Reply:
Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/583/commits/6141692ccb554591eb55543c06df1f77e9a4e959.